app/model/editions/templates/AustralianEdition.scala

```diff
@@ -61,8 +61,8 @@ object AustralianEdition {
   def FrontJournalAu = front(
     "Journal",
     Some("au"),
-    collection("Opinion").searchPrefill("?tag=(australia-news/australia-news,tone/comment)"),
-    collection("World Opinion").searchPrefill("?tag=(tone/comment-australia-news/australia-news)")
+    collection("Opinion").searchPrefill("?tag=(australia-news/australia-news,tone/comment)|tracking/commissioningdesk/australia-opinion"),
+    collection("World Opinion").searchPrefill("?tag=(tone/comment,-australia-news/australia-news)|(tone/comment,-tracking/commissioningdesk/australia-opinion)")
```

Comment:
I think this could be simplified as both the OR elements have tone/comment?  Then I suspect the OR won't produce the effect you want as elements in the second section could be australia news, and elements in the first could be australian opinion?

tone/comment,(-australia-news/australia-news,-tracking/commissioningdesk/australia-opinion)

??